### PR TITLE
1.0.master

### PR DESCRIPTION
**Pull request by gemi2010**
- Ability to list users when viewing a role (role/view)

### CODE REVIEW

**Comment by karelvasicek** [comment]
@gemi2010 , UI is messed up on wide (well, not so wide - 1920px) screens.
obrazek
The `users` box should be below routes in all cases.

**Review by karelvasicek — CHANGES_REQUESTED** [review]
UI must be fixed.

**Comment by karelvasicek** [comment]
@gemi2010 , the grid should have some header too.

File: views/role/view.php [hunk 1]
```diff
@@ -66,6 +68,25 @@
             'style' => 'width:100%']);
         ?>
     </div>
+    <div class="col-lg-11">
```
Comment (karelvasicek) [inline views/role/view.php]: @gemi2010 why did you use `col-lg-11` class?

**Comment by karelvasicek** [comment]
@gemi2010 , the grid header is mispositioned.
obrazek

**Comment by karelvasicek** [comment]
@gemi2010 , one more enhancement - please add button to the grid for opening user record - f.ex. `/admin/assignment/view?id=5`.

**Comment by karelvasicek** [comment]
@gemi2010 , `<h1>` should be used on page only once. The user grid header should be `<h2>` (also to make the grid header a little bit smaller).

**Comment by gemi2010** [comment]
> assignment

done

**Comment by gemi2010** [comment]
> @gemi2010 , `<h1>` should be used on page only once. The user grid header should be `<h2>` (also to make the grid header a little bit smaller).

fixed

**Comment by gemi2010** [comment]
> @gemi2010 , one more enhancement - please add button to the grid for opening user record - f.ex. `/admin/assignment/view?id=5`.

done

**Review by karelvasicek — APPROVED** [review]

**Comment by karelvasicek** [comment]
@gemi2010 , the `<h2>` should have been inside the `<div class="col-lg-12">` of course,  but I meged the PR already and fixed it by myself.

**Pull request merged**